tools/topology/topology2/cavs-sdw-src-gain-mixin.conf

```diff
@@ -137,6 +138,7 @@ Object.Pipeline {
 			Object.Widget.dai-copier.1 {
 				stream_name	$SDW_CAPTURE_PCM
 				dai_type	"ALH"
+				dai_index	10
```

Comment:
is this one right? I don't recall us using a SoundWire DAI '10'?

Reply:
> is this one right? I don't recall us using a SoundWire DAI '10'? 

Yes, this follows the rule from https://github.com/thesofproject/sof/commit/b407a1aab9e40fc060a6b6ec826df9c61512dc8b
Sdw dai index is not used by node_id. The only rule of sdw dai index is don't be duplicated. 
For example, we can't have 2 dai_index 10.

Reply:
@plbossart @bardliao please have a look at this PR again. I have removed those artificial DAI index that we proviously used and replaced it with the actual index of the DAI in an aggregated group. If there is no aggregation the dai_index will always be 0